### PR TITLE
Updated composer.json to support symfony 4

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | none
| License | MIT

#### What's in this PR?

Allows symfony 4 in composer.json

### CODE REVIEW

**Comment by wachterjohannes** [comment]
@alexander-schranz the test fails also in develop so i think we should fix that first

**Comment by alexander-schranz** [comment]
@wachterjohannes you need to update phpunit see https://github.com/sulu/sulu/pull/3931 and reimplement the 2.0 admin for blacklisting feature

**Comment by wachterjohannes** [comment]
@alexander-schranz i have done the phpunit update and the admin is not part of this PR - but symfony is not rendering the `_token` field in the forms (also on develop). do you have an idea why this is happening?

File: composer.json [hunk 1]
```diff
@@ -7,33 +7,34 @@
...
-        "symfony/framework-bundle": "^3.4",
-        "symfony/http-foundation": "^3.4",
-        "symfony/http-kernel": "^3.4",
-        "symfony/intl": "^3.4",
-        "symfony/routing": "^3.4",
-        "symfony/security": "^3.4",
-        "symfony/security-acl": "^2.8 || ^3.0",
-        "symfony/swiftmailer-bundle": "^2.6.4"
+        "massive/build-bundle": "^0.2 || ^0.3 || ^0.4",
+        "sulu/sulu": "dev-develop",
```
Comment (alexander-schranz) [inline composer.json]: keep 2.0

Comment (wachterjohannes) [reply composer.json]: i will change this as soon the alpha 4 is tagged

**Pull request merged**